src/artefacts_demo_control/artefacts_demo_control/collision_test.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import sys
+import os 
+import subprocess
+import json
+import re
+
+import pytest
+import launch_testing
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+from launch_ros.actions import Node
+
+command = ["gz", "model", "-m", "artefacts_box", "-i"]
+result = subprocess.run(command, check = True, text = True, stdout=subprocess.PIPE)
```

Comment:
you will want to make a call to gazebo within a test function, and make sure the robot has finished his movement. So maybe after a certain time

Reply:
Done